allensdk/test/brain_observatory/behavior/conftest.py

```diff
@@ -1,13 +1,16 @@
+import datetime
```

Comment:
there's something non-pythonic about `conftest.py` which is annoying. The fixtures are implicit not explicit. In a test module, I have to hunt around at different levels of `conftest.py` to figure out where a fixture (which I see used but not defined) is defined.
I wonder if [this](https://docs.pytest.org/en/stable/writing_plugins.html#requiring-loading-plugins-in-a-test-module-or-conftest-file) might be a way to more explicitly state what fixtures and their source are being used in test modules.

Reply:
Yeah `conftest.py` can be a pain-point. I guess personally I try to append `_fixture` to all pytest fixtures to make it obvious that a user should look either in the file or `conftest.py`.

The other thing we can request is that if a fixture is not co-located in the same file, there should be a comment specifying where the fixture can be found.

I took a quick read at the plugin page and I'm not sure it would accomplish what we want.

Reply:
ok, just some musing/griping on my part